### PR TITLE
External UCS parameters: Script 007

### PR DESCRIPTION
**Pull request by dboltovskyi**
Script is part of test set: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1169
Common functions for CRQ: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1184

### CODE REVIEW

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT with "disallowed_by_external_consent_entities_off" struct
+-- with invalid type of params
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Accepted

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local sdl = require('SDL')
+local testCasesForExternalUCS = require('user_modules/shared_testcases/testCasesForExternalUCS')
+
+--[[ Local variables ]]
+local grpId = "Location-1"
+local checkedSection = "disallowed_by_external_consent_entities_off"
+local params = { "entityId", "entityType" }
+local values = {
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova :
1) Question "Clarification on max number of elements for disallowed_by_external_consent_entities_on / _off parameter" was raised
2) Accepted

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  { v = "1", desc = "String" },
+  { v = 1.23, desc = "Float" },
+  { v = {}, desc = "Empty table" },
+  { v = { entityType = 1, entityID = 1 }, desc = "Non-empty table" },
+  { v = -1, desc = "Below Min" },
+  { v = 129, desc = "Beyond Max" },
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Please find an answer in clarification "CCS settings limitations"
So possible values are Integers from range [0 .. 128]

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+      end
+      testCasesForExternalUCS.updatePreloadedPT(updateFunc)
+    end
+
+    function Test.StartSDL()
+      StartSDL(config.pathToSDL, config.ExitOnCrash)
+      os.execute("sleep 5")
+    end
+
+    function Test:CheckSDLStatus()
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Solution already implemented in develop branch of ATF
Please find issue "ATF fails test if SDL failed to start"

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 3 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Please check new version of the script 53ca0af

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 2 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Please check new version of the script 53ca0af

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 1 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @istoimenova, Please check new version of the script 53ca0af

Comment (istoimenova) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @dboltovskyi, agree.

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @dboltovskyi, agree.

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @dboltovskyi, thank you for update.

File: test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua (hunk 4 above)
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/007_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua]: @dboltovskyi, thank you for update.

**Pull request merged**